lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -374,9 +365,7 @@ sub _stash_job ($self, $args = undef) {
     return $job;
 }
 
-sub _stash_job_and_module_list {
-    my ($self, $args) = @_;
-
+sub _stash_job_and_module_list ($self, $args) {
```

Comment:
```suggestion
sub _stash_job_and_module_list ($self, $args = undef) {
```

Reply:
Could you see that in the CI tests or just by code analysis? Because from https://app.circleci.com/pipelines/github/os-autoinst/openQA/16073/workflows/af3aef21-37d8-4579-8bca-5276f4bc5042/jobs/154897 that's not obvious at all to me

Reply:
I ran the test locally in verbose mode and got:
```
[error] [wQgCzomItZj0] Too few arguments for subroutine 'OpenQA::WebAPI::Controller::Test::_stash_job_and_module_list' at .../openQA/lib/OpenQA/WebAPI/Controller/Test.pm line 416.
        # Wait for AJAX successful: external results tab for job 99946 loaded
```
I'm actually not sure why it only shows up in verbose mode